### ISSUE TITLE
Rails 4 compatability

### ISSUE THREAD

**Issue by snoblenet** [socialpandas/sidekiq_monitor#13]
On Rails 4 with the `protected_attributes` gem I am getting `ERROR: Can't mass-assign protected attributes for Sidekiq::Monitor::Job: queue, class_name, args, retry, enqueued_at, status, name`

**Comment by thoughtchad** [comment]
@snoblenet - another option would be this: chad-lancour@36b6a83681a0dca9eeb87907f3de2f15579cb1cb

``` ruby
class Job < ActiveRecord::Base
  attr_accessible :args, :class_name, :enqueued_at, :finished_at, :jid, :name, :queue, :result, :retry, :started_at, :status if ActiveRecord::VERSION::MAJOR < 4 || Gem::Specification::find_all_by_name('protected_attributes').any?
```

**Comment by tombenner** [comment]
Fixed in 76fb1ab37c791a936e10e138e7149a05ce310b61.

**Comment by thoughtchad** [comment]
nice
